pkgs/applications/science/biology/poretools/default.nix

```diff
@@ -0,0 +1,23 @@
+{ stdenv, pythonPackages, fetchgit, hdf5 }:
+
+pythonPackages.buildPythonPackage rec {
+  pname = "poretools";
+  version = "0.6.0";
+  name = "${pname}-${version}";
+
+  src = fetchgit {
+    url = "git://github.com/arq5x/${pname}";
+    rev = "e426b1f09e86ac259a00c261c79df91510777407";
+    sha256 = "0bglj833wxpp3cq430p1d3xp085ls221js2y90w7ir2x5ay8l7am";
+  };
+
+  buildInputs = [hdf5];
```

Comment:
Does it need to build against hdf5 that you include it? Otherwise, h5py maintains a reference to hdf5.